packages/block-editor/README.md

```diff
@@ -837,6 +841,33 @@ _Parameters_
 -   _props_ `Object`: Optional. Props to pass to the element. Must contain the ref if one is defined.
 -   _options_ `Object`: Optional. Inner blocks options.
 
+### useLayoutClasses
+
+Generates the utility classnames for the given block's layout attributes.
+
+_Parameters_
+
+-   _blockAttributes_ `Object`: Block attributes.
+-   _blockName_ `string`: Block name.
+
+_Returns_
+
+-   `Array`: Array of CSS classname strings.
+
+### useLayoutStyles
+
+Generates a CSS rule with the given block's layout styles.
+
+_Parameters_
+
+-   _blockAttributes_ `Object`: Block attributes.
+-   _blockName_ `string`: Block name.
+-   _selector_ `string`: A selector to use in generating the CSS rule.
+
+_Returns_
+
+-   `string`: CSS rule.
```

Comment:
For me LayoutStyle, useLayoutStyles and useLayoutClass should be both private APIs based on these. Most blocks shouldn't have to use them to get the layouts working properly.

Reply:
That would be ideal, but given that they have been publicly exposed as `__experimental` already, is it still viable to make them private now? A quick search in [wp directory](https://www.wpdirectory.net/) shows a few hits for each, though they seem to be minified build files so I'm guessing they're just from included packages.

Reply:
I just went ahead and did it anyway 😄 it's unlikely those are actually in use anywhere. In any case, we can revert if there are objections!

Reply:
If it's found that they're used, we could deprecate the public ones for some time, that said, I also think it's very unlikely that any plugin would use these.